src/Pin.js

```diff
@@ -1,110 +1,103 @@
-import React, { PureComponent } from "react";
+import React from "react";
 
-export default class Pin extends PureComponent {
```

Comment:
Why did you update this to no longer be a `PureComponent`? To be fair, I just based this off example code from the map library, so I don't have any particular reason why I did that in the first place. Just want to understand if that was necessary for this fix.

Reply:
It removes an extra dependency and accomplishes the same thing! PureComponent just re-renders on props change no matter what, i.e. a plain ol' function component.